libmscore/tuplet.cpp

```diff
@@ -786,6 +786,8 @@ bool Tuplet::isEditable() const
 
 void Tuplet::startEdit(EditData& ed)
       {
+      //tuplet is not editable with drag now, but still initialize ElementEditData for further possible changes
```

Comment:
Tuplet is editable . I don't understand your comment.

Reply:
Yes, my mistake. I created only full tuplet, so it was not editable. BTW, code is correct, need just remove a string with the comment.